Makefile

```diff
@@ -227,15 +227,15 @@ endif
 .PHONY: install
 install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~/.kube/config.
     ## helm creates objects without aibrix prefix, hence deploying gateway components outside of kustomization
-	$(KUBECTL) create -k config/dependency
+	$(KUBECTL) apply -k config/dependency --server-side
```

Comment:
EG parts looks good to me, we need to use apply --server-side for installing CRDs